tools/Dockerfile

```diff
@@ -1,17 +1,17 @@
-FROM golang:1.19.4
+FROM golang:1.22.3-bookworm
 
 RUN apt-get update && apt-get install -y git
 ENV GOCACHE=/tmp/build/.cache
 ENV GOMODCACHE=/tmp/build/.modcache
-ENV GOFLAGS="-buildvcs=false"
 
 COPY . /tmp/tools
 
 RUN cd /tmp \
   && mkdir -p /tmp/build/.cache \
   && mkdir -p /tmp/build/.modcache \
   && cd /tmp/tools \
-  && go install -trimpath -tags=tools github.com/golangci/golangci-lint/cmd/golangci-lint \
-  && chmod -R 777 /tmp/build/
+  && go install -trimpath github.com/golangci/golangci-lint/cmd/golangci-lint \
+  && chmod -R 777 /tmp/build/ \
+  && git config --global --add safe.directory /build
```

Comment:
I think the `ENV GOFLAGS="-buildvcs=false"` is the actual, preferred fix for the issue? Or did that no longer work?

Reply:
Yeah, I think I was just simply adopting the `Dockerfile` that we use in other `lightninglabs` projects. Modified to use the suggested flag instead.